component/src/main/java/org/wso2/extension/siddhi/io/http/source/util/HttpSourceUtil.java

```diff
@@ -0,0 +1,110 @@
+/*
+ *  Copyright (c) 2017 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+package org.wso2.extension.siddhi.io.http.source.util;
+
+
+import org.apache.log4j.Logger;
+import org.wso2.carbon.transport.http.netty.config.ListenerConfiguration;
+import org.wso2.carbon.transport.http.netty.config.TransportProperty;
+import org.wso2.extension.siddhi.io.http.source.exception.HttpSourceAdaptorRuntimeException;
+import org.wso2.extension.siddhi.io.http.util.HttpConstants;
+import org.wso2.siddhi.core.util.config.ConfigReader;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+
+
+/**
+ * Handles the http source util functions.
+ */
+public class HttpSourceUtil {
+    private static final Logger log = Logger.getLogger(HttpSourceUtil.class);
+
+    public HttpSourceUtil() {
+    }
+
+    /**
+     * Method is responsible for set transportation configuration values.
+     *
+     * @return return the set of netty transportation configuration.
+     */
+    public Set<TransportProperty> getTransportConfigurations(ConfigReader sourceConfigReader) {
+        ArrayList<TransportProperty> properties = new ArrayList<>();
```

Comment:
Declaration should be a list

Reply:
Fixed in 156b93ab839dfc225e0a915f8951b61db7afa2ab